### PR TITLE
Update all missing translation for Indonesian languages.

### PR DESCRIPTION
**Pull request by GID0317**

### CODE REVIEW

**Comment by pfrazee** [comment]
Can we get some confirmations from Indonesian speakers on these translations?

**Comment by GID0317** [comment]
As a native indonesian speaker, i can confirm its good already. but yeah more confirmation from other native indonesian speaker are also welcomed

Buat orang indo yang baca, Silakan kalau misal ada yang kurang atau salah bisa langsung dilist 😉

**Review by thinkbyte1024 — COMMENTED** [review]
Hello there, i've done what I can to review this pull request. Some of the suggestions I've considered are listed below. Other suggestions are in the single comments below.

- The word `edit` needs to be change to `ubah`, since this is a suitable word.
- Some consideration about the word `suggestive` needs to be translated into `sugestif`. As this word is suitable for "safe rating" context.
- There's also a consideration from the previous PR, which is "how do we want to translate the word `feed` or should we keep it?". I would highly suggest to translate this word into `umpan` since our pronunciation is different than English.

There may be something's missing in this review. So I'll check later once this review is resolved. :-)

**Comment by GID0317** [comment]
Thanks for the awesome review! You're right, there are definitely some things that need to be changed. For some translations, I chose the shortest words because longer ones can sometimes break the UI. But if the longer one is okay i guess we can choose the longer one.

File: src/locale/locales/id/messages.po [hunk 1]
```diff
@@ -864,15 +827,15 @@ msgstr "Dibuat {0}"
...
 
 #: src/view/screens/ProfileFeed.tsx:623
 msgid "Created by you"
-msgstr ""
+msgstr "Dibuat oleh Anda"
 
 #: src/view/com/composer/Composer.tsx:448
 msgid "Creates a card with a thumbnail. The card links to {url}"
-msgstr ""
+msgstr "Buat kartu dengan gambar mini. Tautan kartu ke {url}"
```
[the remark this answers is not in the document]
Comment (GID0317) [reply src/locale/locales/id/messages.po:838]: After I give it some thought, is this one instead using a ‘gambar kecil’ or mini for the thumbnail, should we change it to the ‘gambar sampul’ instead?

[the remark this answers is not in the document]
Comment (thinkbyte1024) [reply src/locale/locales/id/messages.po:838]: That would be translated into `cover image`. Perhaps others can suggest an alternative, if not we can stick with `gambar kecil` instead.

[the remark this answers is not in the document]
Comment (GID0317) [reply src/locale/locales/id/messages.po:838]: Okay, i will use the gambar kecil for this one

File: src/locale/locales/id/messages.po [hunk 2]
```diff
@@ -3932,7 +3752,7 @@ msgstr "Ya"
 #: src/view/com/posts/FollowingEmptyState.tsx:67
 #: src/view/com/posts/FollowingEndOfFeed.tsx:68
 msgid "You can also discover new Custom Feeds to follow."
-msgstr ""
+msgstr "Mohon maaf, kami tidak dapat menyelesaikan daftar ini. Jika hal ini terus berlanjut, silakan hubungi pembuat daftar, @{handleOrDid}."
```
Comment (kodebanget) [inline src/locale/locales/id/messages.po:3755]:
```suggestion
msgstr "Anda juga dapat menemukan Feed Khusus baru untuk diikuti."
```

**Review by kodebanget — COMMENTED** [review]
Hello, this is already a good translation. I'm mostly suggesting changes on some inconsistencies and correcting sentence types.

File: src/locale/locales/id/messages.po [hunk 3]
```diff
@@ -1757,100 +1691,95 @@ msgstr "Meninggalkan Bluesky"
...
 
 #: src/view/screens/ProfileList.tsx:323
 msgid "List blocked"
-msgstr ""
+msgstr "Daftar diblokir"
 
 #: src/view/com/feeds/FeedSourceCard.tsx:231
 msgid "List by {0}"
-msgstr ""
+msgstr "Daftar dengan {0}"
```
Comment (oops-wtf) [inline src/locale/locales/id/messages.po:1759]: Screen Shot 2024-01-21 at 16 12 14

```suggestion
msgstr "Daftar oleh {0}"
```

File: src/locale/locales/id/messages.po [hunk 4]
```diff
@@ -1757,100 +1691,95 @@ msgstr "Meninggalkan Bluesky"
...
 
 #: src/view/screens/ProfileFeed.tsx:615
 msgid "Liked by {likeCount} {0}"
-msgstr ""
+msgstr "Disukai oleh {likeCount} {0}"
 
 #: src/view/com/notifications/FeedItem.tsx:171
 msgid "liked your custom feed{0}"
-msgstr ""
+msgstr "menyukai umpan khusus Anda{0}"
```
Comment (oops-wtf) [inline src/locale/locales/id/messages.po:1731]:
```suggestion
msgstr "menyukai feed Anda{0}"
```

**Review by thinkbyte1024 — COMMENTED** [review]
Is there any chance that we want to translate the word `feed` into `umpan`? I've seen some words already translated into `umpan`, such as in line 1731.

File: src/locale/locales/id/messages.po [hunk 5]
```diff
@@ -1104,20 +1063,19 @@ msgstr "Edit Feed Tersimpan"
 
 #: src/view/com/modals/CreateOrEditList.tsx:188
 msgid "Edit User List"
-msgstr ""
+msgstr "Edit Daftar Pengguna"
```
[the remark this answers is not in the document]
Comment (thinkbyte1024) [reply src/locale/locales/id/messages.po:1066]: Personally, I would go with the word ubah. But if the word ubah would mean changing something in a larger scope, then perhaps we could go with edit.

**Review by oops-wtf — COMMENTED** [review]
Small improvement and all...

Hi @ansh isn't it time to move to mature platforms like Crowdin for doing this kind of thing?

**Comment by pfrazee** [comment]
Excellent, thank you everybody. Merging now.

> Hi @ansh isn't it time to move to mature platforms like Crowdin for doing this kind of thing?

It's on my todos to review this

> Aside from the suggestions above, they seem OK 👍

Awesome, do you speak Indonesian?

**Pull request merged**